core/src/Platforms/iOS/WebviewBase.cs

```diff
@@ -40,7 +40,7 @@ internal abstract class WebviewBase : NSObject, IWebUI, ISFSafariViewControllerD
         protected static UIViewController viewController;
         protected SFSafariViewController safariViewController;
         protected SFAuthenticationSession sfAuthenticationSession;
-        protected ASWebAuthenticationSession asWebAuthenticationSession;
+        //protected ASWebAuthenticationSession asWebAuthenticationSession;
```

Comment:
> //protected ASWebAuthenticationSession asWebAuthenticationSession; [](start = 7, length = 67)

not good

Reply:
this should not have been checked in. reverting